benefits/core/widgets.py

```diff
@@ -15,3 +16,26 @@ def __init__(self, pattern=None, placeholder=None, **kwargs):
             self.attrs.update({"pattern": pattern})
         if placeholder:
             self.attrs.update({"placeholder": placeholder})
+
+
+class RadioSelect(widgets.RadioSelect):
+    template_name = "core/widgets/radio_select.html"
+    option_template_name = "core/widgets/radio_select_option.html"
+
+    def __init__(self, choice_descriptions=(), *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.choice_descriptions = list(choice_descriptions)
+
+    def __deepcopy__(self, memo):
+        obj = super().__deepcopy__(memo)
+        obj.choice_descriptions = copy.copy(self.choice_descriptions)
+        return obj
```

Comment:
Since our `RadioSelect` subclass introduces `choice_descriptions` as an instance variable, we want to make sure that when anyone makes a deep copy of a `RadioSelect` object, that copy will have its own instance of `choice_descriptions` rather than sharing the instance from the original `RadioSelect`. (The call to `super().__deepcopy__(memo)` on line 30 is handling the other instance variables, of course.)

Notably, when Django instantiates [forms](https://github.com/django/django/blob/main/django/forms/forms.py#L107) and their [fields](https://github.com/django/django/blob/main/django/forms/fields.py#L142), it uses deep-copying, so there would presumably be unexpected behavior / bugs if we didn't have this.

These Django tickets have examples of the kind of unexpected behavior that would occur:
- https://code.djangoproject.com/ticket/25085
- https://code.djangoproject.com/ticket/12048

Reply:
(To be fair, our current usage of `RadioSelect` wouldn't expose the bug.)

Reply:
@angela-tran Thanks for this background info! 🙏